### PR TITLE
EVG-13105: do not store URL with test results in S3

### PR DESCRIPTION
**Pull request by Kimchelly**
JIRA: https://jira.mongodb.org/browse/EVG-13105

Compute the log URL only for the REST route.

### CODE REVIEW

File: rest/model/test_results.go [hunk 1]
```diff
@@ -28,7 +31,7 @@ func (a *APITestResult) Import(i interface{}) error {
 		a.TestName = ToAPIString(tr.TestName)
 		a.Trial = tr.Trial
 		a.Status = ToAPIString(tr.Status)
-		a.LogURL = ToAPIString(tr.LogURL)
+		a.LogURL = ToAPIString(strings.Join([]string{"test_log", tr.TaskID, strconv.Itoa(tr.Execution), tr.TestName}, "/"))
```
Comment (julianedwards) [inline rest/model/test_results.go:34]: This URL should be: `<cedar_host>/rest/v1/buildlogger/test_name/{task_id}/{test_name}?execution={execution}`.

File: test_results.proto [hunk 2]
```diff
@@ -26,7 +26,6 @@ message TestResult {
   string test_name = 1;
   int32 trial = 2;
   string status = 3;
-  string log_url = 4;
```
Comment (julianedwards) [inline test_results.proto:29]: Fix the numbers here, we are now skipping 4.

**Review by julianedwards — CHANGES_REQUESTED** [review]

File: rest/model/test_results.go (hunk 1 above)
Comment (Kimchelly) [reply rest/model/test_results.go]: Fixed.

File: test_results.proto (hunk 2 above)
Comment (Kimchelly) [reply test_results.proto:29]: Done.

**Review by julianedwards — APPROVED** [review]

**Pull request merged**